packages/communication-react/review/communication-react.api.md

```diff
@@ -684,6 +709,9 @@ export type CustomMessagePayload = {
 // @public
 export const darkTheme: PartialTheme & CallingTheme;
 
+// @public (undocumented)
+export type DefaultCallControlButton = 'microphone' | 'camera' | 'screenShare' | 'participants' | 'leave';
```

Comment:
nit: Name this `DefaultCallControlButtons` or `DefaultCallControlButtonTypes`, or anything to distinguish from `CustomCallControlButton`. One is a string literal type, while the other is an interface.